Makefile

```diff
@@ -0,0 +1,14 @@
+default: serve
+
+build:
+	docker build . -t social-web
```

Comment:
What about `sdp-web` or `sdp-www`? I suggest using `sdp` as a prefix or namespace for this and other images so when I list all my docker stuff I can immediately recognize what's for `sdp`.

Reply:
that is true i regularly confuse my social protocols, i shall include this change in my next update